bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOTest.java

```diff
@@ -81,8 +70,6 @@ public void testInitialize() throws Exception{
     checkRegistry(Put.class);
```

Comment:
Coder registry is mocked. I think we should keep it in unit testing.
"when(underTest.getCoderRegistry()).thenReturn(registry);"